.babelrc

```diff
@@ -3,13 +3,8 @@
     [
       "env", {
         "targets": {
-          "node": "6"
+          "node": "current"
```

Comment:
All the Node stuff are just because Node 12 is working so fucking fast on my machine...